### PR TITLE
Template app generator

### PR DESCRIPTION
**Pull request by j8seangel**
See [readme](https://github.com/GlobalFishingWatch/frontend/compare/template-app-generator?expand=0#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R36) to know how to use the generator

### CODE REVIEW

File: tools/generators/app/files/project.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      },
+      "configurations": {
+        "production": {}
+      }
+    },
+    "start": {
+      "executor": "@nrwl/workspace:run-commands",
+      "options": {
+        "commands": [
+          "nx clean-locales port",
```
Comment (rdgfuentes) [inline tools/generators/app/files/project.json:44]: Instead of `port` this should be the `<%= name %>`

**Review by rdgfuentes — APPROVED** [review]
This is a great start point for bootstrapping new apps @j8seangel! Great work! 👏

File: tools/generators/app/lib/add-project.ts [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    // a missing configuration error will be thrown.
+    configurations: {
+      production: {},
+    },
+  }
+
+  targets.start = {
+    executor: '@nrwl/workspace:run-commands',
+    options: {
+      commands: ['nx clean-locales port', 'nx serve i18n-labels', 'nx serve port'],
```
Comment (rdgfuentes) [inline tools/generators/app/lib/add-project.ts:52]: Same here, instead of `port` it should be `${options.projectName}`, right?

**Comment by fertandil87** [comment]
🤓

**Pull request merged**